internal/export/batcher.go

```diff
@@ -51,7 +51,7 @@ func (s *Server) handleCreateBatches() http.Handler {
 			if errors.Is(err, database.ErrAlreadyLocked) {
 				stats.Record(ctx, mBatcherLockContention.M(1))
 				logger.Infow("already locked")
-				w.WriteHeader(200)
+				w.WriteHeader(http.StatusOK)
```

Comment:
I actually have a note to go through and make helpers for these and always return JSON.

200 is the right code though I think. It means someone else is already running. We don't want Cloud Scheduler to retry in this case.

Reply:
Agreed on that one (although maybe we should comment that it's for scheduler - in normal REST world I might expect 409 StatusConflict or something).

Reply:
but what about the above in the export batcher enum?

Reply:
right, this is 200 so the CS doesn't report failure, I know it's a bit weird